templates/web/src/sdk.ts.twig

```diff
@@ -13,6 +13,7 @@ namespace Models {
          * {{ property.description }}
          */
         {{ property.name | escapeKeyword }}{% if not property.required %}?{% endif %}: {{_self.sub_schema(property)}};
+        {% if definition.additionalProperties %}[key: string]?: any;{% endif %}
```

Comment:
actually, this needs to be outside of the loop - otherwise it will create multiple index types 😉

Reply:
also I think this is the correct syntax

```suggestion
        {% if definition.additionalProperties %}[key: string]: any;{% endif %}
```

Reply:
Done